Tests/BSGConnectivityTest.m

```diff
@@ -138,7 +139,7 @@ - (void)mockMonitorURLWithCallback:(BSGConnectivityChangeBlock)block {
 }
 
 - (void)simulateConnectivityChangeTo:(SCNetworkReachabilityFlags) flags {
-    BSGConnectivityCallback(nil, flags, nil);
+    BSGConnectivityCallback(nil, flags, "Unit Test");
```

Comment:
Discard file changes - no longer required

Reply:
Leaving in since we now have a null-guard in test callbacks.

Reply:
This is probably a good place to document the mechanism we're using in a comment